Formula/libxmlsec1.rb

```diff
@@ -43,6 +44,11 @@ def install
             "--prefix=#{prefix}",
             "--disable-crypto-dl",
             "--disable-apps-crypto-dl",
+            "--with-libxslt=no",
```

Comment:
This might be used on macOS.

Reply:
If it is used on macOS, it should be added as an explicit dependency.

Reply:
Not necessarily, since what is used would be the macOS-provided `libxslt`. Declaring `uses_from_macos` isn't actually for macOS (these lines do nothing there), but for Linux.

Reply:
Would removing
```rb
"--with-libxslt=no",
```
and adding
```rb
  on_linux do
    depends_on "libxslt"
  end
```
be OK?

Reply:
I went ahead and made this change

Reply:
Indeed, this is used on macOS:
```
❯ brew linkage libxmlsec1
System libraries:
[...]
  /usr/lib/libxslt.1.dylib
[...]
```
so deactivating it would be a regression. Not sure we should be adding this on Linux, but ok.